tests/Basic/CookieTest.php

```diff
@@ -95,6 +95,42 @@ public function cookieWithPathsDataProvider()
         );
     }
 
+    /**
+     * @dataProvider cookieWithPathsDataProvider
+     */
+    public function testCookieInSubPath($cookieRemovalMode)
+    {
+        // Start clean.
+        // The cookie is set when viewing the page.
+        $session = $this->getSession();
+        $session->visit($this->pathTo('/sub-folder/cookie_page2.php'));
+        $this->assertContains('Previous cookie: NO', $session->getPage()->getText());
+
+        $session->visit($this->pathTo('/sub-folder/cookie_page4.php'));
+
+        // On the next load, the cookie has been set.
+        $session->visit($this->pathTo('/sub-folder/cookie_page2.php'));
+        $this->assertContains('Basic Form Previous cookie: srv_var_is_set', $session->getPage()->getText());
```

Comment:
Please remove `Basic Form ` (note space at the end) so that assertion would be similar to other cookie tests.

The `Basic Form` is getting in the output in this driver, but not in others (e.g. `Selenium2`).